tests/mikrotik_routeros/ip_dhcp-server_lease_print_terse_without-paging/mikrotik_routeros_ip_dhcp-server_lease_print_terse_without-paging_2.yml

```diff
@@ -1,59 +1,82 @@
 ---
 parsed_sample:
-  - index: "0"
-    flag: ""
-    comment: "UniFi1"
+  - active_address: "10.164.1.199"
+    active_client_id: "1:74:83:c2:36:5a:6e"
+    active_mac_address: "74:83:C2:36:5A:6E"
+    active_server: "dhcp1"
     address: "10.124.3.199"
-    mac_address: "74:83:C2:36:5A:6E"
-    client_id: "1:74:83:c2:36:5a:6e"
     address_lists: ""
-    server: "dhcp1"
+    age: ""
     always_broadcast: ""
+    client_id: "1:74:83:c2:36:5a:6e"
+    comment: "UniFi1"
     dhcp_option: "unifi"
-    status: "bound"
     expires_after: "6m51s"
-    last_seen: "3m9s"
-    active_address: "10.164.1.199"
-    active_mac_address: "74:83:C2:36:5A:6E"
-    active_client_id: "1:74:83:c2:36:5a:6e"
-    active_server: "dhcp1"
+    flag: ""
     host_name: "Unifi3B"
+    index: "0"
+    last_seen: "3m9s"
+    mac_address: "74:83:C2:36:5A:6E"
+    server: "dhcp1"
     src_mac_address: ""
-  - index: "1"
-    flag: ""
-    comment: "PC"
+    status: "bound"
+  - active_address: "10.164.1.252"
+    active_client_id: "1:0:50:56:88:80:ee"
+    active_mac_address: "00:50:56:88:80:EE"
+    active_server: "dhcp1"
     address: "10.124.3.252"
-    mac_address: "00:50:56:88:80:EE"
-    client_id: "1:0:50:56:88:80:ee"
     address_lists: ""
-    server: "dhcp1"
+    age: ""
     always_broadcast: ""
+    client_id: "1:0:50:56:88:80:ee"
+    comment: "PC"
     dhcp_option: ""
-    status: "bound"
     expires_after: "6m18s"
-    last_seen: "3m42s"
-    active_address: "10.164.1.252"
-    active_mac_address: "00:50:56:88:80:EE"
-    active_client_id: "1:0:50:56:88:80:ee"
-    active_server: "dhcp1"
+    flag: ""
     host_name: "RSNC-CLIENT1-VM"
+    index: "1"
+    last_seen: "3m42s"
+    mac_address: "00:50:56:88:80:EE"
+    server: "dhcp1"
     src_mac_address: ""
-  - index: "2"
-    flag: ""
-    comment: "SPA"
+    status: "bound"
+  - active_address: "10.164.1.222"
+    active_client_id: "1:c4:a:cb:bc:65:c8"
+    active_mac_address: "C4:0A:CB:BC:65:C8"
+    active_server: "dhcp1"
     address: "10.124.3.222"
-    mac_address: "C4:0A:CB:BC:65:C8"
-    client_id: ""
     address_lists: ""
-    server: "dhcp1"
+    age: ""
     always_broadcast: ""
+    client_id: ""
+    comment: "SPA"
     dhcp_option: ""
-    status: "bound"
     expires_after: "5m56s"
-    last_seen: "4m4s"
-    active_address: "10.164.1.222"
-    active_mac_address: "C4:0A:CB:BC:65:C8"
-    active_client_id: "1:c4:a:cb:bc:65:c8"
-    active_server: "dhcp1"
+    flag: ""
     host_name: "SPA122"
+    index: "2"
+    last_seen: "4m4s"
+    mac_address: "C4:0A:CB:BC:65:C8"
+    server: "dhcp1"
+    src_mac_address: ""
+    status: "bound"
+  - active_address: "10.0.1.56"
+    active_client_id: "1:c2:39:b3:fa:81:d0"
+    active_mac_address: "C2:39:B3:FA:81:D0"
+    active_server: "Clients"
+    address: "10.0.1.56"
+    address_lists: ""
+    age: "2h11m23s"
```

Comment:
Phew! :grinning:
One of the entries has an `age`. We have test coverage. :sweat_smile: